### PR TITLE
Remove RAJA::forallN

### PR DESCRIPTION
**Pull request by ajkunen**
Not really much to say.... except that it's gone

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
😢

**Comment by ajkunen** [comment]
@davidbeckingsale Ok, it looks good now.

File: examples/waveShared.cpp [hunk 1]
```diff
@@ -111,9 +111,19 @@ void innerLoop(const serialLoop &, int Ny, int Nx, Func &&innerLoop){
   }
 #endif
```
Comment (artv3) [inline examples/waveShared.cpp]: @ajkunen , can we delete this file?

Comment (ajkunen) [reply examples/waveShared.cpp]: @artv3 , yeah, those were your examples... I started converting one of them, but then realized they weren't even in the CMakeLists files anymore.
Do you want to remove them as an ancillary PR?

**Pull request merged**